### PR TITLE
Release v1.1.0

### PR DESCRIPTION
**Pull request by Gauthamastro**
# Changelog

### Added

- Github workflow to publish binary on release ([#54])

### Changed

- Upgrade of Substrate stack dependencies to `0.9.43` ([#58])
- Used Thea pallet for crypto operations ([#59])

[#54]: https://github.com/Polkadex-Substrate/parachain/pull/54
[#58]: https://github.com/Polkadex-Substrate/parachain/pull/58
[#59]: https://github.com/Polkadex-Substrate/parachain/pull/59
[v1.1.0]: https://github.com/Polkadex-Substrate/parachain/compare/v1.0.0...v1.1.0

### CODE REVIEW

**Comment by serhii-temchenko** [comment]
FYI @Gauthamastro and @zktony I changed title of this PR to `Release v1.1.0` since `Release v1.0.0` is already taken.
Plus I raised 2 PR's, one with [CI changes ](https://github.com/Polkadex-Substrate/parachain/pull/62) and one with [release notes update](https://github.com/Polkadex-Substrate/parachain/pull/63). Please check them and after merging them - this PR could be merged and release created.

**Comment by zktony** [comment]
LGTM. You create the release. I will upload the artefacts into it.

**Comment by serhii-temchenko** [comment]
> LGTM. You create the release. I will upload the artefacts into it.

Ok. And no need to create artifcts for it manually I think if you are about release binary since it should be done automatically after merge of PR #54. This will be the first time in this repo so we will test it on release :)

File: runtime/src/lib.rs
Comment (35359595) [inline runtime/src/lib.rs:180]: should we jump 3 versions for release? (probably incremented for testnet)

File: runtime/src/lib.rs
Comment (35359595) [inline runtime/src/lib.rs:470]: isn't this a DDOS enabler?

File: runtime/src/lib.rs
Comment (35359595) [inline runtime/src/lib.rs:404]: isn't this DDOS enabler?

File: runtime/src/lib.rs
Comment (35359595) [inline runtime/src/lib.rs:403]: should this be nothing?

File: runtime/src/lib.rs
Comment (35359595) [inline runtime/src/lib.rs:354]: why these two are nothing?
`HoldIdentifier` in [substrate](https://github.com/paritytech/substrate/blob/5e49f6e44820affccaf517fd22af564f4b495d40/bin/node/runtime/src/lib.rs#L465) 
`FreezeIdentifier` is () in substrate but should it be same here?

**Review by 35359595 — CHANGES_REQUESTED** [review]

**Review by 35359595 — APPROVED** [review]

File: runtime/src/lib.rs
Comment (zktony) [reply runtime/src/lib.rs]: Current parachain version is same

File: runtime/src/lib.rs
Comment (zktony) [reply runtime/src/lib.rs:404]: Acala and astar are using same setting

File: runtime/src/lib.rs
Comment (zktony) [reply runtime/src/lib.rs:403]: acala and astar are using same config

File: runtime/src/lib.rs
Comment (zktony) [reply runtime/src/lib.rs:470]: Yeah .. I created new issue for this https://github.com/Polkadex-Substrate/parachain/issues/64

File: runtime/src/lib.rs
Comment (zktony) [reply runtime/src/lib.rs:354]: We are not using these features in parachain. But I will update it.

**Comment by serhii-temchenko** [comment]
@Gauthamastro, @zktony, @35359595 as you can see in the latest CI run, we have errors and I think this is because of new compiler version. I tried to resolve problem (compilation errors) by bumping substrate version to `polkadot-v1.0.0` but no result. Could someone check it so we could roll out a new release?

Versions info:

> rustup 1.26.0 (5af9b9484 2023-04-05)
> info: The currently active `rustc` version is `rustc 1.73.0-nightly (1b198b3a1 2023-08-13)`
> rustfmt 1.6.0-nightly (1b198b3 2023-08-13)
> clippy 0.1.73 (1b198b3 2023-08-13)

**Comment by serhii-temchenko** [comment]
> @Gauthamastro, @zktony, @35359595 as you can see in the latest CI run, we have errors and I think this is because of new compiler version. I tried to resolve problem (compilation errors) by bumping substrate version to `polkadot-v1.0.0` (because previous version was released before new compiler release so no support in it for sure) but no positive result. Could someone check it so we could roll out a new release?
> 
> Versions info:
> 
> > rustup 1.26.0 (5af9b9484 2023-04-05)
> > info: The currently active `rustc` version is `rustc 1.73.0-nightly (1b198b3a1 2023-08-13)`
> > rustfmt 1.6.0-nightly (1b198b3 2023-08-13)
> > clippy 0.1.73 (1b198b3 2023-08-13)

To not hold this release any longer - I raised PR #65 with pinned compiler version where everything compiles properly. Please take a look.

**Comment by Gauthamastro** [comment]
> > @Gauthamastro, @zktony, @35359595 as you can see in the latest CI run, we have errors and I think this is because of new compiler version. I tried to resolve problem (compilation errors) by bumping substrate version to `polkadot-v1.0.0` (because previous version was released before new compiler release so no support in it for sure) but no positive result. Could someone check it so we could roll out a new release?
> > Versions info:
> > > rustup 1.26.0 (5af9b9484 2023-04-05)
> > > info: The currently active `rustc` version is `rustc 1.73.0-nightly (1b198b3a1 2023-08-13)`
> > > rustfmt 1.6.0-nightly (1b198b3 2023-08-13)
> > > clippy 0.1.73 (1b198b3 2023-08-13)
> 
> To not hold this release any longer - I raised PR #65 with pinned compiler version where everything compiles properly. Please take a look.

This is better than upgrading to v1.0.0, we need state migrations and other stuff for it.

**Pull request merged**
